diff --git a/builder/pep517.py b/python/builder/pep517.py
index e91df98..9977f51 100644
--- a/builder/pep517.py
+++ b/builder/pep517.py
@@ -30,12 +30,12 @@ def get_requires_for_build_wheel(config_settings=None):
     # set up version constraints: note that CalVer like 22.03 is normalized to
     # 22.3 by setuptools, so we must follow the same practice in the constraints;
     # also, we don't need the patch number here
-    cuqnt_require = [f'custatevec-cu{utils.cuda_major_ver}~=1.8',   # ">=1.8.0,<2"
-                     f'cutensornet-cu{utils.cuda_major_ver}~=2.7',   # ">=2.7.0,<3"
-                     f'cudensitymat-cu{utils.cuda_major_ver}~=0.1'  # ">=0.1, <0.2"
-                    ]
-
-    return _build_meta.get_requires_for_build_wheel(config_settings) + cuqnt_require
+    #cuqnt_require = [f'custatevec-cu{utils.cuda_major_ver}~=1.8',   # ">=1.8.0,<2"
+    #                 f'cutensornet-cu{utils.cuda_major_ver}~=2.7',   # ">=2.7.0,<3"
+    #                 f'cudensitymat-cu{utils.cuda_major_ver}~=0.1'  # ">=0.1, <0.2"
+    #                ]
+    #return _build_meta.get_requires_for_build_wheel(config_settings) + cuqnt_require
+    return _build_meta.get_requires_for_build_wheel(config_settings)
 
 
 # Note: We have never promised to support sdist (CUQNT-514). We really cannot
diff --git a/cuquantum/bindings/_internal/cudensitymat_linux.pyx b/python/cuquantum/bindings/_internal/cudensitymat_linux.pyx
index 6e26258..0954f6b 100644
--- a/cuquantum/bindings/_internal/cudensitymat_linux.pyx
+++ b/cuquantum/bindings/_internal/cudensitymat_linux.pyx
@@ -90,7 +90,7 @@ cdef void* __cudensitymatWorkspaceGetMemory = NULL
 
 cdef void* load_library() except* nogil:
     cdef void* handle
-    handle = dlopen("libcudensitymat.so.0", RTLD_NOW | RTLD_GLOBAL)
+    handle = dlopen("@cuda_bundle@/lib/libcudensitymat.so.0", RTLD_NOW | RTLD_GLOBAL)
     if handle == NULL:
         with gil:
             err_msg = dlerror()
diff --git a/cuquantum/bindings/_internal/custatevec_linux.pyx b/python/cuquantum/bindings/_internal/custatevec_linux.pyx
index 176471a..947cee0 100644
--- a/cuquantum/bindings/_internal/custatevec_linux.pyx
+++ b/cuquantum/bindings/_internal/custatevec_linux.pyx
@@ -113,7 +113,7 @@ cdef void* __custatevecGetMathMode = NULL
 
 cdef void* load_library() except* nogil:
     cdef void* handle
-    handle = dlopen("libcustatevec.so.1", RTLD_NOW | RTLD_GLOBAL)
+    handle = dlopen("@cuda_bundle@/lib/libcustatevec.so.1", RTLD_NOW | RTLD_GLOBAL)
     if handle == NULL:
         with gil:
             err_msg = dlerror()
diff --git a/cuquantum/bindings/_internal/cutensornet_linux.pyx b/python/cuquantum/bindings/_internal/cutensornet_linux.pyx
index a68e879..49ba2ab 100644
--- a/cuquantum/bindings/_internal/cutensornet_linux.pyx
+++ b/cuquantum/bindings/_internal/cutensornet_linux.pyx
@@ -155,7 +155,7 @@ cdef void* __cutensornetStateApplyGeneralChannel = NULL
 
 cdef void* load_library() except* nogil:
     cdef void* handle
-    handle = dlopen("libcutensornet.so.2", RTLD_NOW | RTLD_GLOBAL)
+    handle = dlopen("@cuda_bundle@/lib/libcutensornet.so.2", RTLD_NOW | RTLD_GLOBAL)
     if handle == NULL:
         with gil:
             err_msg = dlerror()
diff --git a/setup.py b/python/setup.py
index 3964a9f..3bc8332 100644
--- a/setup.py
+++ b/setup.py
@@ -35,9 +35,9 @@ with open(os.path.join(source_root, "tests/requirements.txt")) as f:
 install_requires = [
     'numpy>=1.21, <3.0',  # ">=1.21,<3"
     # 'torch', # <-- PyTorch is optional; also, the PyPI version does not support GPU...
-    f'custatevec-cu{utils.cuda_major_ver}~=1.8',   # ">=1.8.0,<2"
-    f'cutensornet-cu{utils.cuda_major_ver}~=2.7',  # ">=2.7.0,<3"
-    f'cudensitymat-cu{utils.cuda_major_ver}~=0.1', # ">=0.1,<0.2"
+    #f'custatevec-cu{utils.cuda_major_ver}~=1.8',   # ">=1.8.0,<2"
+    #f'cutensornet-cu{utils.cuda_major_ver}~=2.7',  # ">=2.7.0,<3"
+    #f'cudensitymat-cu{utils.cuda_major_ver}~=0.1', # ">=0.1,<0.2"
 ]
 if utils.cuda_major_ver == '11':
     install_requires.append('cupy-cuda11x>=13.0')  # no ambiguity
